src/drivers/uavcan/sensors/battery.hpp

```diff
@@ -85,5 +96,11 @@ class UavcanBatteryBridge : public UavcanSensorBridgeBase, public ModuleParams
 	uint8_t _warning;
 	hrt_abstime _last_timestamp;
 	battery_status_s _battery_status[battery_status_s::MAX_INSTANCES] {};
-	bool battery_aux_support[battery_status_s::MAX_INSTANCES] {false};
+	BatteryDataType _batt_update_mod[battery_status_s::MAX_INSTANCES] {BatteryDataType::Raw};
```

Comment:
Be aware that this initializes only the first to your value, the rest to 0. Better use `BatteryDataType _batt_update_mod[battery_status_s::MAX_INSTANCES] {};` to avoid surprises.